docs/DEVGUIDE.md

```diff
@@ -183,10 +183,12 @@ Use Helm to create the Kubernetes deployments:
 
     export VERSION=$(git describe --tags --always --abbrev=7 --dirty)
     helm install \
-        --set "registry=${REGISTRY},version=${VERSION}" \
+        --set "registry=${REGISTRY},version=${VERSION},storageType=${STORAGE_TYPE}" \
         --namespace catalog \
         ./deploy/catalog
 
+Note that we are using an environment variable that we didn't set `STORAGE_TYPE`, with a null value it will default to `etcd`.
```

Comment:
This is a weird comment. I get the gist of it but if we're not going to tell people to set it to a value why bother to include it at all?  Can we either not add it to the DEVGUIDE at this time, or tell people they can:
1 - leave it off (and it defaults to 'etcd')
2 - set it to "etcd"
3 - set if to ???? what? TPR?